drivers/Ofek.py

```diff
@@ -1,8 +1,127 @@
 from rose.common import obstacles, actions  # NOQA
 
-driver_name = "Sidan"
+driver_name = "Stern"
+
+
+def obstacle(world, x, y):
+    try:
+        obstacle = world.get((x, y))
+    except IndexError:
+        # x, y - 1 is not a valid position
```

Comment:
```suggestion
        # x, y is not a valid position
```
it's actually checking for x,y now (not x, y-1 like before the change)